src/amo/components/HeroRecommendation/index.js

```diff
@@ -60,6 +62,41 @@ export const addParamsToHeroURL = ({
 };
 
 export class HeroRecommendationBase extends React.Component<InternalProps> {
+  callToActionURL: string;
+
+  static defaultProps = {
+    _tracking: tracking,
+  };
+
+  constructor(props: InternalProps) {
+    super(props);
+
+    const { shelfData } = props;
+    invariant(shelfData, 'The shelfData property is required');
+
+    const { addon, external } = shelfData;
+    this.callToActionURL = '';
+
+    if (addon) {
+      this.callToActionURL = addParamsToHeroURL({
+        urlString: `/addon/${addon.slug}/`,
```

Comment:
It would be nice if we had a shared function for this. This isn't a change request, just a comment. I see it hard coded in `SearchResult`, `AddonReviewList`, `AddonSummaryCard`, and `AddonTitle` 👀

Reply:
Thanks @kumar303, I opened #8563 for this.